docs/index.md

```diff
@@ -5,13 +5,17 @@ title: RAVE AddIn
 Welcome to the Riverscapes Analysis Visualization Explorer (RAVE) software website. Here you can find general information about the RAVE software, including information on [opening projects](open-project.html), [symbolizing layers](symbology.html) and controlling how projects are displayed using [business logic XML](business-logic.html).
 
 <div style="text-align:center">
+<a class="button large" href="http://rave.riverscapes.xyz/install.html#qgis-plug-in">
+        <img src="{{site.baseurl}}/assets/images/RiverscapesLogo_x16.png">
+        &nbsp;&nbsp;Installation Instructions for QRAVE PlugIn for QGIS</a>
 <a class="button large" href="https://github.com/Riverscapes/RaveAddIn/releases/latest">
         <img src="{{site.baseurl}}/assets/images/RiverscapesLogo_x16.png">
-        &nbsp;&nbsp;Download RAVE</a>
+        &nbsp;&nbsp;Download ArcRAVE AddIn for ArcGIS 10.X</a>
+<a class="button large" href="https://data.riverscapes.xyz">
+        <img src="{{site.baseurl}}/assets/images/RiverscapesLogo_x16.png">
+        &nbsp;&nbsp;Explore WebRAVE in Warehouse/a>
 </div>
 
-![RAVE]({{ site.baseurl}}/assets/images/rave.png)
-
 ## Background
 
 RAVE AddIn is part of the [Riverscapes Consortium's](http://riverscapes.xyz) much larger family of tools for analyzing riverscapes. RAVE itself does not do any analysis, but allows exploring of any [Riverscapes-compliant project](https://riverscapes.github.io/riverscapes-website/Technical_Reference/Documentation_Standards/Riverscapes_Projects/). Rave provides an explorer in which you can navigate the "project tree", and add any layer to map symbolized appropriately. 
```

Comment:
Update this text so it is consistent about RAVE (spelling) and not sometimes referring to RAVEAdd In.